src/design-system/typography/typeHierarchy.ts

```diff
@@ -1,6 +1,36 @@
+/**
+ * ⚠️ IMPORTANT NOTE ⚠️
```

Comment:
Maybe also add a link to the Figma spec here so devs can visually reference what is in this file?

Reply:
I'm not sure how set in stone the Figma URL is, so I might come back to this one.